### PR TITLE
doc for alert-avatar-badge-button

### PR DESCRIPTION
**Pull request by srishtigautam21**
added the code snippets for
avatar
alert
badge
button

### CODE REVIEW

File: css/components.css [hunk 1]
```diff
@@ -9,8 +9,10 @@
 @import url('../component/Image/image.css');
 @import url('../component/Input/input.css');
 @import url('../css/landingPage.css');
+@import url('../css/docs.css');
 @import url('../css/style.css');
 
+
 :root{
     /* --primary-color:#22c55e; */
     /* --primary-color:#16a34a; */
```
Comment (zoef-yt) [inline css/components.css]: remove commented code.

File: css/docs.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.content-header{
+    font-size: 4.8rem;
+    font-weight: 900;
+    line-height: 6.4rem;
+    letter-spacing: 1px;
+    /* text-align: left; */
```
Comment (zoef-yt) [inline css/docs.css]: remove commented code

File: css/docs.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    color:var(--COLOR-BLACK);
+    margin-bottom:1rem;
+}
+.md-text{
+    /* font-size: rem; */
```
Comment (zoef-yt) [inline css/docs.css]: remove commented code

File: css/style.css [hunk 4]
```diff
@@ -64,20 +64,23 @@
 .content-container{
     display: flex;
     flex-direction: column;
+    /* display: block; */
     /* justify-content: center; */
     /* align-items: center; */
     /* margin:5rem 5rem 5rem 30rem; */
```
Comment (zoef-yt) [inline css/style.css]: remove commented code

File: css/style.css [hunk 5]
```diff
@@ -64,20 +64,23 @@
...
     height: 100%;
     width:24.8rem;
     overflow-x: hidden;
     padding-top: 2rem;
-    border-right: 1px solid var(--COLOR-SOCIAL-ICON);
+    margin-left:2rem;
+    /* border-right: 1px solid var(--COLOR-SOCIAL-ICON); */
     /* position: fixed;
     left:0;
     top:8.8rem; */
```
Comment (zoef-yt) [inline css/style.css]: remove commented code

File: css/style.css [hunk 6]
```diff
@@ -105,6 +108,7 @@
 .main-container{
     display:flex;
     flex-wrap: wrap;
+    width:100%;
     /* margin-left:30rem;
     margin-top: 12rem; */
```
Comment (zoef-yt) [inline css/style.css]: remove commented code

**Review by zoef-yt — CHANGES_REQUESTED** [review]
remove commented code.

File: component/Badges/badge.css [hunk 7]
```diff
@@ -44,7 +44,7 @@
     border-radius: 50%;
     cursor: pointer;
 }
-.status-badge{
+.avatar-status{
     position:absolute;
     width:14px;
```
Comment (developer-anand) [inline component/Badges/badge.css:49]: you can use rem here

**Review by developer-anand — APPROVED** [review]
good to go with changes.

File: component/Badges/badge.css (hunk 7 above)
Comment (srishtigautam21) [reply component/Badges/badge.css:49]: oh i forgot i had to change them thanks for pointing it out i will do it

File: component/Avatar/avatar.css [hunk 8]
```diff
@@ -25,29 +26,29 @@
...
 }
-.xsmall-text-avatar{
+.xs-text-avatar{
     background-color:var(--COLOR-PRIMARY);
     font-size: 1.3rem;
     font-weight: 600;
     text-align: center;
     padding-top:6px;
-    margin:4.9rem 0rem 0rem 0rem;
+    margin:5.5rem 0rem 0rem 0rem;
```
Comment (iRohitGaur) [inline component/Avatar/avatar.css:51]: .md-text-avatar, .sm-text-avatar and .xs-text-avatar have a lot of common properties. you can combine them in one class.

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: component/Avatar/avatar.css (hunk 8 above)
Comment (srishtigautam21) [reply component/Avatar/avatar.css:51]: ok will look into it

**Review by zoef-yt — APPROVED** [review]
Good to go. Good to merge.

**Pull request merged**